lib/getAffiliateMintEvents.tsx

```diff
@@ -0,0 +1,24 @@
+import { BigNumber, utils } from "ethers"
+import { getAddress } from "ethers/lib/utils"
+import { ethGetLogs } from "./alchemy/eth_getLogs"
+
+export const getAffiliateMintEvents = async () => {
+  const eventSignature = utils.id("AffiliateSale(address,uint256,uint256,uint256)")
```

Comment:
you change the emit right? This needs to be updated

Reply:
updated ✅